build.gradle

```diff
@@ -70,8 +66,6 @@ dependencies {
     compile group: 'org.jboss.modules', name: 'jboss-modules', version: '1.1.0.GA'
 
     compileOnly group: 'javax.activation', name: 'activation', version: '1.1.1'
-    compileOnly group: 'log4j', name: 'log4j', version: '1.2.16'
```

Comment:
Curious what was happening that required this

Reply:
A different version of log4j is getting pulled in as a dependency, creating a conflict.  Found out by using the dependency hell jar in assembly.